packages/autorest.python/autorest/codegen/models/base.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, yaml_data: Dict[str, Any], code_model: "CodeModel") -> None:
         self.api_versions: List[str] = yaml_data.get(
             "apiVersions", []
         )  # api versions this type is in.
+        self.enable_generate: bool = yaml_data.get("enableGenerate", True)
```

Comment:
is the only use case for `enableGenerate` content type enums? if so it mght be cleaner just to have something in enum type that says "i won't generate conteent type enums". right now it's hard to tell when and wheree this is used and what it means

Reply:
I move it to enum_type.py and add annotation